src/ARToolkit.Maui/Handlers/ARSceneViewHandler.cs

```diff
@@ -14,21 +14,24 @@
 //  *   limitations under the License.
 //  ******************************************************************************/
 
+using Esri.ArcGISRuntime.Maui;
+using Esri.ArcGISRuntime.Maui.Handlers;
+using Esri.ArcGISRuntime.UI;
 using Microsoft.Maui.Handlers;
 
 namespace Esri.ArcGISRuntime.ARToolkit.Maui.Handlers
 {
     public class ARSceneViewHandler
- #if WINDOWS || __IOS__ || __ANDROID__
-         : ViewHandler<IARSceneView, Esri.ArcGISRuntime.ARToolkit.ARSceneView>
- #else
+#if WINDOWS || __IOS__ || __ANDROID__
+         : ArcGISRuntime.Maui.Handlers.SceneViewHandler
+#else
```

Comment:
This doesn't feel quite right. We need `IARSceneView` in the viewhandler interface.